Tools/TestWebKitAPI/Tests/WebKitCocoa/TextManipulation.mm

```diff
@@ -3012,6 +3012,48 @@ - (void)_webView:(WKWebView *)webView didFindTextManipulationItem:(_WKTextManipu
     TestWebKitAPI::Util::run(&done);
 }
 
+TEST(TextManipulation, CompleteTextManipulationDoesNotSkipTabCharacterAtLineWrap)
+{
+    auto delegate = adoptNS([[TextManipulationDelegate alloc] init]);
```

Comment:
Nit: I think we prefer RetainPtr to auto nowadays

Reply:
ok